src/games/strategy/triplea/Constants.java

```diff
@@ -223,6 +223,8 @@
   public static final String ABANDONED_TERRITORIES_MAY_BE_TAKEN_OVER_IMMEDIATELY =
       "Abandoned Territories May Be Taken Over Immediately";
   public static final String DISABLED_PLAYERS_ASSETS_DELETED = "Disabled Players Assets Deleted";
+  public static final String CONTROL_ALL_CANALS_BETWEEN_TERRITORIES_TO_PASS =
```

Comment:
Can you comment if it is worth moving these string constants one day to Properties.java? The string constants generally are used for 2 reasons: 
- DRY
- avoid magic strings, ie: assign labels to specific values, so values can be referenced by label rather than by the value itself.

In this case, I don't think we get either one. The function that uses it makes it very clear what the string would be, so it seems like we are just adding an unnecessary layer of indirection.

Would you agree that would be a good future refactor?